libs/widget-lib/src/types.ts

```diff
@@ -204,11 +204,6 @@ export interface CowSwapWidgetParams {
    */
   theme?: CowSwapTheme | CowSwapWidgetPalette
 
-  /**
-   * Allows to set a custom logo for the widget.
-   */
-  logoUrl?: string
```

Comment:
We never display logo in the widget

Reply:
I guess we will need to update the docs too once we are done with the changes https://docs.cow.fi/cow-protocol/tutorials/widget